packages/client/src/index.ts

```diff
@@ -82,6 +82,17 @@ const provider = new CCIPReadProvider(baseProvider);
     const responseData = await resolver.resolve(dnsName(name), data);
     const addr = Resolver.decodeFunctionResult('addr(bytes32)', responseData);
     console.log(`${name}: ${addr}`);
+
+    const textData = Resolver.encodeFunctionData('text(bytes32,string)', [
+      node,
+      'description',
+    ]);
+    const textResponseData = await resolver.resolve(dnsName(name), textData);
+    const value = Resolver.decodeFunctionResult(
+      'text(bytes32,string)',
+      textResponseData
+    );
+    console.log(`${name}: ${value}`);
```

Comment:
Should modify both of these log statements to state what they're logging - eg, 'addr()' and 'text(description)'.